### DISCUSSION TITLE
Hip joint ideas

### DISCUSSION THREAD

**Discussion by Chompalot** [makerforgetech/modular-biped#59] [Ideas]
I've modelled a rough setup as a starter to improve mobility.
The cardle with the 2 servos is roughly the same size as the current build, so you can see that there are issues getting the servos in.
The leg is the standard build

![image001](https://github.com/dmt-labs/modular-biped/assets/34891324/1be90157-8f6d-4141-b265-dbe2182c0a35)

**Comment by danic85** [comment]
Nice! :clap:

Yeah the servos are a bit bulky, it has been a problem with the whole design. It might be worth calculating the load on the servos, as the MG92Bs have a decent stall torque, and might work instead (or another micro servo alternative).
